src/main/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBBase.java

```diff
@@ -263,6 +271,26 @@ public ListGlobalTablesResult listGlobalTables(ListGlobalTablesRequest listGloba
         throw new UnsupportedOperationException("not yet supported");
     }
 
+    @Override
+    public DescribeGlobalTableSettingsResult describeGlobalTableSettings(DescribeGlobalTableSettingsRequest describeGlobalTableSettingsRequest) {
+        throw new UnsupportedOperationException("not yet supported");
```

Comment:
Do we plan to support them? If not, consider leaving out "yet". (Similarly with similar things below...)